src/drivers/pca9685_pwm_out/main.cpp

```diff
@@ -101,14 +100,11 @@ class PWMDriverWrapper : public cdev::CDev, public ModuleBase<PWMDriverWrapper>,
 
 	uORB::Subscription _parameter_update_sub{ORB_ID(parameter_update)}; // param handle
 
-	MixingOutput _mixing_output{PCA9685_MAX_OUTPUT_CHANNELS, *this, MixingOutput::SchedulingPolicy::Auto, true};
-
-private:
-
+	MixingOutput _mixing_output{PCA9685_PWM_CHANNEL_COUNT, *this, MixingOutput::SchedulingPolicy::Auto, true};
 };
 
 PWMDriverWrapper::PWMDriverWrapper() :
-	CDev(PCA9685_DEVICE_BASE_PATH),
+	CDev(nullptr),
```

Comment:
Needs review. I found it safe to pass nullptr for that I want to get dynamic allocation of CDev path to avoid conflicts.

Reply:
Yes you can get away with this.